docs/github.md

```diff
@@ -67,7 +67,7 @@ changes.
 ## Working in a Git co-located repository
 
 After doing `jj git init --colocate`, Git will be in a [detached HEAD
-state][detached], which is unusual, as Git mainly works with bookmarks. In a
+state][detached], which is unusual, as jj mainly works with bookmarks. In a
```

Comment:
Considering the first commit in isolation, I think this ought to be `as Git mainly works with branches`, rather than `as jj mainly works with bookmarks`. I believe this sentence is meant to expand on the fact that "detached HEAD state" is unusual for Git itself.

Reply:
Someone who has used Git might have learned that being in a detached HEAD state is dangerous or a mistake. I think this section's goal is to reassure that person that it's okay, and jj will record any commits you make in Git as long as you run `jj` before you do any `git checkout` or `git switch` commands (is that right? I never mutate repos with git tooling so I don't actually know).

Reply:
@arxanas Good observation. I have updated the changes with that in mind.

@jennings Good point. May I suggest that tackling that is out of scope here? At this point I'm just trying to get the facts right :)